frontend/src/components/footer.css

```diff
@@ -1,8 +1,14 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Montserrat', sans-serif;
+}
 .footer {
   margin-top: auto;
   bottom: 0;
-  height: 100px;
+  height: 132px;
```

Comment:
Why are we setting a specific height here, rather than having this be responsive with the page size/viewing device?